lib/3scale/core/errors.rb

```diff
@@ -126,5 +126,11 @@ def initialize(error_text)
         super %(ServiceToken is invalid, #{error_text})
       end
     end
+
+    class UsersRequiresServiceId < Error
```

Comment:
it's already defined a few lines above: https://github.com/3scale/pisoni/pull/15/commits/a26d89d86295cabcac6370d4c4759bb86b577f0f#diff-a761ded015e35c580d7119a41d7ed7c2R64

Reply:
That's for `User` class. We are defining for `Users` class.